src/VisualStudio/IntegrationTest/TestUtilities/VisualStudioInstanceFactory.cs

```diff
@@ -272,12 +278,94 @@ private static Process StartNewVisualStudioProcess(string installationPath)
             IntegrationHelper.KillProcess("dexplore");
 
             var process = Process.Start(vsExeFile, VsLaunchArgs);
-
             Debug.WriteLine($"Launched a new instance of Visual Studio. (ID: {process.Id})");
 
             return process;
         }
 
+        // Starts ProcDump against the processId.
```

Comment:
yes, will do

---
In reply to: [121030956](https://github.com/dotnet/roslyn/pull/20129#discussion_r121030956) [](ancestors = 121030956)